### ISSUE TITLE
krbtgt account not filtered out

### ISSUE THREAD

**Issue by bandrel** [skelsec/kerberoast#13]
krbtgt account not filtered out when pulling kerberoast hashes. Its unlikely that the krbtgt would be manually set and therefore somewhat of waisted cycles which is why most tools exclude it from the output.

**Comment by skelsec** [comment]
yeah.. this project is sporadically maintained due to `minikerberos` library contains kerberoasting code and I started using that a year ago, and the minikerberos version this project targeted originally was filtering krbtgt our in the other module.  
Nevertheless thank you for notifying me I'm in the process of organizing my projects and will update this one in the next cycle. (just look at the PRs XD)

**Comment by bandrel** [comment]
I just added the issue incase anyone else tried to crack an krbtgt on accident.
